test.cpp

```diff
@@ -45,7 +45,7 @@ static int assertEquals(const std::string &expected, const std::string &actual,
     if (expected != actual) {
         numberOfFailedAssertions++;
         std::cerr << "------ assertion failed ---------" << std::endl;
-        std::cerr << "line " << line << std::endl;
+        std::cerr << "line test.cpp:" << line << std::endl;
```

Comment:
This allows some IDE to show a clickable link making it easier to navigate to the failure.